packages/website/src/features/geocoder/lib/utilities.test.tsx

```diff
@@ -0,0 +1,568 @@
+import "@testing-library/jest-dom";
+import { forwardGeocodeNeighborhoodCouncil } from "./utilities";
+
+describe("Forward Geocode Neighborhood Council", () => {
+  it("should return an empty array for a query string that does not match any neighborhood council", () => {
+    const result = forwardGeocodeNeighborhoodCouncil("abc");
+    expect(result).toEqual([]);
+  }),
+    it("should return the same results for a query string regardless of case", () => {
+      const result1 = forwardGeocodeNeighborhoodCouncil("downtown los angeles");
+      const result2 = forwardGeocodeNeighborhoodCouncil("Downtown Los Angeles");
+      expect(result1).toEqual(result2);
+    }),
+    it("should return neighborhood councils that partially match the query", () => {
+      const result1 = forwardGeocodeNeighborhoodCouncil("Downtown Los");
+      const result2 = forwardGeocodeNeighborhoodCouncil("Downtown Los Angeles");
+      expect(result1).toEqual(result2);
+    }),
+    it("should return an array of GeocodeResult type for input query string", () => {
+      const result = forwardGeocodeNeighborhoodCouncil("Downtown Los Angeles");
+      expect(result).toEqual([
+        {
```

Comment:
nit. This is a large test object. We can put test/dummy date in another file. We can do this another time.